CHANGELOG.md

```diff
@@ -1,21 +1,26 @@
 # CHANGELOG
 
+## UNRELEASED
+
+* Added Swift 3 support (Xcode 8 beta 2) while retaining Swift 2 support.
```

Comment:
Please add two spaces at the end of each line description (before the newline) so that the rendered markdown adds a newline (instead of keeping the next line in the same line when rendered).

> https://daringfireball.net/projects/markdown/syntax#p
> When you do want to insert a <br /> break tag using Markdown, you end a line with two or more spaces, then type return.